packages/flutter/lib/src/widgets/table.dart

```diff
@@ -371,16 +372,23 @@ class _TableElement extends RenderObjectElement {
   @override
   void insertRenderObjectChild(RenderObject child, dynamic slot) {
     renderObject.setupParentData(child);
+    if (_lastRemovedParentDataInThisFrame != null) {
```

Comment:
I have analyzed all the scenarios of inserting RO, it is safe to do so and can fix this bug

Reply:
There are three situations where this function will be called,
1. In the `_TableElement`  mount phase, `_lastRemovedParentDataInThisFrame ` is null at this time;
2. The `_TableElement ` update phase. For one row update, the `void update(Table newWidget)` will first insert the new RO, then remove the old RO, and then update the next Row. The first RO inserted coordinations must be the last deletion of the previous row. The Updata will also call `_updateRenderObjectChildren` at the end, so the operation is safe.
3. In the scene of the issue problem, `_lastRemovedParentDataInThisFrame ` will record the last delete the coordinates of RO. If there is an insertion in this frame, we can find the position to be inserted.